requirements/requirements_nlp.txt

```diff
@@ -5,6 +5,7 @@ fasttext
 flask_restful
 ftfy
 gdown
+git+https://github.com/NVIDIA/lddl.git
```

Comment:
Remove and make optional until we have a pip package.

Reply:
removed import and added instructions to launcher for optional import